package.json

```diff
@@ -13,7 +13,8 @@
     "dev": "npm start --prefix .dev/",
     "build": "tsc && npm run copy-css",
     "copy-css": "copyfiles -u 1 \"src/extensions/**/*.css\" dist",
-    "publish": "npm publish --access public "
+    "publish": "npm publish --access public",
```

Comment:
I wonder if it's actually worth publishing the plugins to the npm registry before they are really final. It's pretty straightforward to `npm install` them from Github. At least during the dev/testing stage, this might speed things up a bit.